task-manager-backend/src/task/controllers/get-all-tasks/get-all-tasks.js

```diff
@@ -0,0 +1,23 @@
+const ApiResponse = require("../../../utils/api-response");
+const ApiError = require("../../../utils/api-error");
+const { getAllTasksUsecase } = require("../../usecases");
+
+const { TASK_FETCH_SUCCESS } = require("../../../commons/constants");
+
+module.exports = {
+    getAllTasks: {
+        path: "/",
```

Comment:
this should beuser/:userId /tasks

Reply:
same here, I have added "/users" in app, did I need to remove that?